tests/demo_tests/test_ip_v4v6_raw_capture.py

```diff
@@ -0,0 +1,196 @@
+import pytest
+import snappi
+
```

Comment:
move this test to `raw` dir and call it `test_raw_tcp_flow_capture.py` (since that's what we're doing here)

Reply:
addressed